src/wdio/services/TerraCommands/chai-methods.js

```diff
@@ -8,10 +8,23 @@ function accessible() {
   // eslint-disable-next-line no-underscore-dangle
   new chai.Assertion(this._obj).to.be.instanceof(Array);
 
+  // These are the new rules introduced in axe-core v3.3 and v3.4 to disable for now to remain passive.
+  const axeRuleIdsToDisable = [
+    'aria-input-field-name',
+    'aria-roledescription',
+    'aria-toggle-field-name',
+    'avoid-inline-spacing',
+    'input-button-name',
+    'landmark-unique',
+    'role-img-alt',
+    'scrollable-region-focusable'
+  ];
+
   // eslint-disable-next-line no-underscore-dangle
   const errors = this._obj
     .filter(test => test.result)
     .reduce((all, test) => all.concat(test.result.violations), [])
+    .filter(test => !axeRuleIdsToDisable.includes(test.id))
```

Comment:
Once we enable this, teams will be surprised by the number of rules they may need to fix. Can we check for errors that were caused by these rules and potential log a warning so coming work is visible ?